### PR TITLE
Add about page

### PR DESCRIPTION
**Pull request by steffendsommer**
This should cover "About" and "Contact".

### CODE REVIEW

File: Resources/Views/pages/about.leaf [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                    <h4 class="f-weight-sm">
+                        ServerSide.swift is a framework-independent conference, where we will learn and share on a number of different related topics.
+                        The conference is aimed at being a non-profit conference and solely run for the love of server-side Swift. This space is still new and we hope to be able to contribute to the growth of ServerSide.swift by facilitating a conference where people can meet and collaborate. We have all been following and taking part in the server-side Swift movement and after running meetups in Berlin and London, we believe that the community is ready for a conference dedicated to server-side Swift.
+                    </h4>
+                    <h4 class="f-weight-sm">
+                        We want people to feel safe and welcome at the conference and we expect all attendees to respect our <a href="/code-of-conduct">Code of Conduct</a>.
+                    </h4>
+
+                    <h3>The Team</h3>
+                    <h4 class="f-weight-sm">
```
Comment (MartinLasek) [inline Resources/Views/pages/about.leaf]: I guess this can be removed 😊 ?

File: Resources/Views/pages/about.leaf [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                    <h3>The Conference</h3>
+                    <h4 class="f-weight-sm">
+                        ServerSide.swift is a framework-independent conference, where we will learn and share on a number of different related topics.
+                        The conference is aimed at being a non-profit conference and solely run for the love of server-side Swift. This space is still new and we hope to be able to contribute to the growth of ServerSide.swift by facilitating a conference where people can meet and collaborate. We have all been following and taking part in the server-side Swift movement and after running meetups in Berlin and London, we believe that the community is ready for a conference dedicated to server-side Swift.
+                    </h4>
+                    <h4 class="f-weight-sm">
+                        We want people to feel safe and welcome at the conference and we expect all attendees to respect our <a href="/code-of-conduct">Code of Conduct</a>.
+                    </h4>
+
+                    <h3>The Team</h3>
```
Comment (MartinLasek) [inline Resources/Views/pages/about.leaf]: I would wrap it into a `<div class="col-xs-12"></div>` and put it within the next `<div class="row">` right before all the `<div class="col-xs-12 col-sm-4 col-md-4">` columns of our images come. 😊

File: Resources/Views/pages/about.leaf [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                    <h4 class="f-weight-sm">
+
+                    </h4>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-xs-12 col-sm-4 col-md-4">
```
Comment (MartinLasek) [inline Resources/Views/pages/about.leaf]: If the column behaviour stays the same for larger screen sizes there's no need to specify it. 😊 

TL;DR You can remove the `col-md-4` 😊

File: Resources/Views/pages/about.leaf [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                                    #embed("partials/icons/twitter")
+                                </a>
+                            </div>
+                        </div>
+                        <div class="card">
+                            <div class="title">
+                                Martin Lasek
+                            </div>
+                            <div class="sub-title">
+                                Indie game developer & Swift Developer
```
Comment (MartinLasek) [inline Resources/Views/pages/about.leaf]: How about `Indie Game Developer, Vapor Tutorial Writer & Swift Developer` 😊 ?

File: Resources/Views/pages/about.leaf [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+                    </div>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-xs-12">
+        
+                    <h3>Getting in touch</h3>
+                    <h4 class="f-weight-sm">
+                        For general questions and requests, please use <a href="mailto:info@serversideswift.info">info@serversideswift.info</a>. For anything related to sponsoring the conference, please reach out to us at <a href="mailto:sponsors@serversideswift.info">sponsors@serversideswift.info</a>. If you're interested in speaking, please submit a talk through out CFP form <a href="https://goo.gl/forms/Jb37ZKROJNm5HIGl2" target="_blank">here</a>. You are also welcome to reach out to us through the social media channels at the bottom or using our individual Twitter handles.
```
Comment (MartinLasek) [inline Resources/Views/pages/about.leaf]: Found one minor thing: through out => throughout 😊

File: Public/styles/about.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    background-image: -moz-linear-gradient(left, #F88A36 0%, #FD2020 100%), -moz-linear-gradient(left, #F88A36 0%, #FD2020 100%);
+    background-image: -o-linear-gradient(left, #F88A36 0%, #FD2020 100%), -o-linear-gradient(left, #F88A36 0%, #FD2020 100%);
+    background-image: linear-gradient(to right, #F88A36 0%, #FD2020 100%), linear-gradient(to right, #F88A36 0%, #FD2020 100%);
+}
+.organizer .card .title {
+    font-size: 22px;
+}
+
+.organizer {
+    padding: 50px;
```
Comment (MartinLasek) [inline Public/styles/about.css]: I'd suggest to change it to `padding: 0 50px;` to get the spacing between headers and content consistent 😊
